src/langcheck/augment/_common/_jailbreak_template.py

```diff
@@ -14,6 +15,7 @@ def jailbreak_template_common(
     num_perturbations: int = 1,
     randomize_order: bool = True,
     seed: int | None = None,
+    custom_templates: list[tuple[str, str]] | None = None,
```

Comment:
I like this design, which allows us handling the custom templates in the similar way as we do for the built-in templates now!!